web/src/js/components/FlowView/Connection.tsx

```diff
@@ -8,9 +8,9 @@ type ConnectionInfoProps = {
 
 export function formatAddress(
     desc: string,
-    address: Address | undefined,
+    address: Address | undefined | null,
 ): React.ReactElement {
-    if (address === undefined) {
+    if (address === undefined || address === null) {
```

Comment:
```suggestion
    if (!address) {
```
(in contrast to the flow reducer changes, here we don't care about the difference at all)